### ISSUE TITLE
File loading error

### ISSUE THREAD

**Issue by faroit** [marl/medleydb#15]
#14 seems to break the track loading

a simple

```
mtrack_list = mdb.load_all_multitracks()

for track in mtrack_list:
    print track
```

now results in:

```
(.env)$ py test.py
Traceback (most recent call last):
  File "medleydb/utils.py", line 62, in load_multitracks
    yield M.MultiTrack(multitrack)
  File "medleydb/multitrack.py", line 75, in __init__
    self.title = _path_basedir(mtrack_path).split('_')[1]
IndexError: list index out of range
```

**Comment by rabitt** [comment]
I think it's time for some continuous build pipelines eh? 
Looking into this...

**Comment by rabitt** [comment]
This is fixed now in v1.1
